android/app/src/main/java/com/integreat/MainApplication.java

```diff
@@ -3,9 +3,10 @@
 import android.app.Application;
 
 import com.facebook.react.ReactApplication;
+import com.swmansion.gesturehandler.react.RNGestureHandlerPackage;
```

Comment:
For what do we need the GestureHandler atm?

Reply:
This is a new dependency of some package. not sure which though.